src/EPR.Payment.Service.Common.Data/Migrations/AppDbContextModelSnapshot.cs

```diff
@@ -749,6 +789,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                             Id = 7,
                             Description = "Re-submitting a report",
                             Type = "ReSubmitting"
+                        },
+                        new
+                        {
+                            Id = 8,
+                            Description = "Producer Late Registration Fee",
```

Comment:
I think the Description should be "Late fee", and the Type "LateFee". This way the same subtype can be used for Producers and Compliance schemes. There will be no need to use a separate subtype for either.